services/job_runner_test.go

```diff
@@ -283,29 +281,6 @@ func TestJobRunner_savesOverridesOnError(t *testing.T) {
 	assert.Equal(t, initialData, run.Overrides.Data)
 }
 
-func TestJobRunner_errorsOnOverride(t *testing.T) {
```

Comment:
I think this may be a required test case related to my comment ^ about L305-306

Reply:
Ahh, I'll revisit.

Reply:
This has been moved to `TestJobRunsController_Update_WithMergeError`